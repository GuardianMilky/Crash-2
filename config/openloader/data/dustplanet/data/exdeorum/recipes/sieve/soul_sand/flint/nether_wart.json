{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:soul_sand"},"mesh":"exdeorum:flint_mesh","result":"minecraft:nether_wart","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.06}}