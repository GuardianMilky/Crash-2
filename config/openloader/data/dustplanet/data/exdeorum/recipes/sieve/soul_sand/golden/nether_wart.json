{"type":"exdeorum:sieve","ingredient":{"item":"minecraft:soul_sand"},"mesh":"exdeorum:golden_mesh","result":"minecraft:nether_wart","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.08}}